core/geometry/src/polyface/IndexedEdgeMatcher.ts

```diff
@@ -12,7 +12,7 @@
  * * Fixed entry positions are:
  *   * [0] is start vertex index (in CCW order around its facet)
  *   * [1] is end vertex index (in CCW order around its facet)
- *   * [2] is facet index.
+ *   * [2] is "facet index".
```

Comment:
not sure if quote mark is needed OR you should add quote for [0] and [1] as well

Reply:
To be addressed in next commit.